first_order_model/config/paper_configs/personalization/resolution512_our_model.yaml

```diff
@@ -1,5 +1,5 @@
 dataset_params:
```

Comment:
Please do not change any of my existing files in paper_configs, please revert this and use a different file insteadPlease do not change any of my existing files in paper_configs, please revert this and use a different file instead